modulefiles/ufs_gaea_c5.intel.lua

```diff
@@ -0,0 +1,33 @@
+help([[
+  This module loads libraries required for building and running UFS Weather Model 
+  on the NOAA RDHPC machine Gaea C5 using Intel-2023.1.0.
+]])
+
+whatis([===[Loads libraries needed for building the UFS Weather Model on Gaea ]===])
+
+load("PrgEnv-intel/8.3.3")
+load("intel-classic/2023.1.0")
+load("cray-mpich/8.1.25")
+load("python/3.9.12")
+
+prepend_path("MODULEPATH", "/lustre/f2/dev/wpo/role.epic/contrib/spack-stack/c5/spack-stack-dev-20230717/envs/ufs-pio-2.5.10/install/modulefiles/Core")
```

Comment:
Is there a non-dev spack stack available to switch to permanently ?

Reply:
this is the only version available on c5 right now. it is not necessarily a "develop" version, just built a specific spack-stack develop hash that contained necessary changes not present in the 1.4.0 release branch.

Reply:
@ulmononian I've been testing on C5, and I was able to use

``/lustre/f2/dev/wpo/role.epic/contrib/spack-stack/spack-stack-1.4.1-c5``

Reply:
i should have been clear: there are other stacks on c5, but the stack installed to address all issues on c5 that were encountered with spack-stack 1.4.1 is `/lustre/f2/dev/wpo/role.epic/contrib/spack-stack/c5/spack-stack-dev-20230717/envs/ufs-pio-2.5.10`. please see https://spack-stack.readthedocs.io/en/latest/PreConfiguredSites.html#noaa-rdhpcs-gaea-c5.

in any case, spack-stack 1.5.0 will installed on c5 in the near term.